setup.cfg

```diff
@@ -1,5 +1,6 @@
 [flake8]
```

Comment:
Project is shifting towards black formatting. I would recommend the same here, skipping flake8.

Reply:
I agree that it would be good to start using black (which already is done via a githook) but I prefer to do that in a separate ticket.